### DISCUSSION TITLE
Transpile to Vala?

### DISCUSSION THREAD

**Discussion by ghost** [pfusik/cito#123] [General]
The language is syntactically similar to C# so I think it will be easy for you to implement.

**Comment by pfusik** [comment]
I've heard about Vala, but never used it. It seems to be aimed at Linux GUI apps?

It's not the syntax that matters. Python's different syntax wasn't a problem. Swift's value semantics of arrays and lack of increment/decrement operators were problematic.

**Reply by ghost** [reply]
You could read more about Vala on wiki: https://en.wikipedia.org/wiki/Vala_(programming_language)

**Reply by pfusik** [reply]
Thanks, I've read it.

**Reply by ghost** [reply]
What do you think about transpiling to Vala?

**Reply by pfusik** [reply]
Probably good idea, but low priority. There are many more popular languages to transpile to.
